spec/features/availability_spec.rb

```diff
@@ -66,6 +66,18 @@
       expect(page).to have_selector 'strong', text: /Being acquired by the library/
     end
 
+    xit 'that has a public note' do
```

Comment:
@banukutlu - I'm going to revisit the work I started a while ago in improving these unit tests. We're starting to add even more pending specs. 😭

Reply:
ahhhh yeah! availability specs always 😬